defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+sansible_pyenv:
+  user: root
+  activate: no
+  python_version: system
```

Comment:
Hmm.. We generally set vars that require configuring to "~"
Would it be better to set this to ~ and check that "sansible_pyenv.python_version is not none" in your when clauses?

Reply:
I can see two options here:

1) Fail right at the beginning if python_version isn't defined
2) a set_fact to defaults if user, activate, or python_version isn't defined

Reply:
Default is now none and I am asserting a version has been selected first thing.